gamestonk_terminal/etf/etf_controller.py

```diff
@@ -4,19 +4,21 @@
 import argparse
 import os
 from typing import List
+
 import matplotlib.pyplot as plt
+import pandas as pd
 from prompt_toolkit.completion import NestedCompleter
+
 from gamestonk_terminal import feature_flags as gtff
-from gamestonk_terminal.helper_funcs import get_flair
-from gamestonk_terminal.menu import session
-from gamestonk_terminal.etf.stockanalysis_model import (
-    name_search,
-    open_web,
-    etf_overview,
-    compare_etfs,
-    etf_holdings,
+from gamestonk_terminal.helper_funcs import (
+    get_flair,
+    parse_known_args_and_warn,
+    export_data,
 )
-from gamestonk_terminal.etf.screener_model import etf_screener
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.etf import stockanalysis_view
+from gamestonk_terminal.etf import stockanalysis_model
```

Comment:
let's try to not import models in the controllers.

Reply:
So I am not sure how to get around this one.  Since i use the model to define all possible etfs ,which is assigned to self.  That then gets used to make sure inputs are valid.  Either I import there or do gymnastics creating new type functions just to get around importing that.

Reply:
So, I would have that list on the view for instance, to make it easier

Reply:
But I was thinking, and it can potentially be necessary having _model here. We can discuss this during this week

Reply:
Yeah I think in cases it will be necessary.  At least in this menu, if I don't do it here, I have to call that load function within each `view` function.  If I define it outside of the functions, then it gets called/loaded on import, which we have decided should not be done.

Reply:
Yup, I agree.